### DISCUSSION TITLE
[Resolved] Valid Student Developer Pack Approval

### DISCUSSION THREAD

**Discussion by ghost** [community/community#17783] [GitHub Education]
While submitting my 2022 Graduation PR, I get an error saying 

"I'm not seeing a valid student developer pack approval, please submit an [application](https://education.github.com/discount_requests/student_application?utm_source=2021-06-05-GitHubGraduation) to get started"

But I do have an active Student Developer Pack as seen in the 2 screenshots below.

![image](https://user-images.githubusercontent.com/62909766/166432703-47e445e2-cb67-47d4-a2ee-d3ef43f822a6.png)
![image](https://user-images.githubusercontent.com/62909766/166432758-d8503d40-d433-4262-a4c2-652a588b307e.png)

Unsure who to contact for help.

**Comment by mlarasusan** [comment]
Hello @roeegh , how did you solve this problem?

**Reply by ghost** [reply]
Hey @mlarasusan, I was able to resolve it by re-requesting a review a few hours after my initial commit.
